docs/src/2.4.ManagingTheCamera.mdx

```diff
@@ -17,7 +17,7 @@ In this step, we'll add a duck model to the center of the scene, and move the ca
 First, let's add a duck to the scene using [GLTFScene](#/docs/commands/gltfscene), and add the `Duck.glb` model file to the `src/utils` folder.
 
 ```js
-import duckModel from "common/fixtures/Duck.glb"; // Webpack magic: we actually import a URL pointing to a .glb file
+import duckModel from "~/common/fixtures/Duck.glb"; // Webpack magic: we actually import a URL pointing to a .glb file
```

Comment:
I assume this means something other than "import from your $HOME directory"

Reply:
yeah, it's configured in webpack resolve.alias. I stole this idea from @defunctzombie

Reply:
The previous state of affairs was that the whole project root directory was being used as a path to resolve modules, which I didn't like